.github/workflows/integration_tests.yml

```
@@ -0,0 +1,83 @@
+  pull_request:
+    paths:
+      - '**/*login*.*'
+  workflow_dispatch:  # Allows manual triggering
+
+jobs:
+  integration_test_android:
+    name: Integration Tests on Android
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v3
```

actionlint (1.7.4)

the runner of "actions/checkout@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)